docs/oadp_cheat_sheat.md

```diff
@@ -0,0 +1,224 @@
+[![hackmd-github-sync-badge](https://hackmd.io/6-i34q7uR_2um-yH1trmbA/badge)](https://hackmd.io/6-i34q7uR_2um-yH1trmbA)
```

Comment:
Simultaneous editing by maintainers can be achieved within GitHub PR, not sure if it's necessary to duplicate effort to hackmd and have to then PR it back here.

Reply:
true... I'll turn off the hackmd sync